docs/2.0/docs/patcher/tutorials/applying-first-patch.md

```diff
@@ -1,3 +1,161 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+```
+
+Now we'll run `patcher update`.  This can be run from the root of the repository, where it will scan every unit in all sub-folders, or directly inside the <CustomizableValue id="DIRECTORY" /> directory.
+
+```bash
+
+$ cd $$DIRECTORY$$
+$ patcher update
+```
+
+
+![Patcher Update homepage showing an update to the patcher-test module is available](/img/patcher/tutorials/patcher_update_preview.png)
+
+From here you can hit enter to update `patcher-test`, or `u` to view usages of the module, `v` to view the changelog, or `q` to quit.
+
+<details>
+<summary>Hitting `u` to view changes shows the following:</summary>
+
+![The changes page shows every unit that uses the module, and what the most recent version it](/img/patcher/tutorials/patcher_update_usages.png)
+</details>
+
+<details>
+<summary>Hitting `v` to view the changelog shows the following:</summary>
+
+![The changelog page shows the changelog directly from the upstream module](/img/patcher/tutorials/patcher_update_changelog.png)
+</details>
+
+From here hit `enter` and patcher will apply the update:
+
+![Patcher update shows that it found a patch to be applied and what version it is incrementing to](/img/patcher/tutorials/patcher_update_in_progress.png)
+
+![Patcher Update completion notice](/img/patcher/tutorials/patcher_update_complete.png)
+
+You can now inspect the filesystem and see the results of the patch.  In this case it resulted in changing the version of `patcher-test` from `v0.10.3` to `v0.10.5` and added a new required argument `sampleinput` to the unit. This change can now be committed and pushed to your repository.
+
+![Patcher Update completion notice](/img/patcher/tutorials/patcher_update_results.png)
+
+## Applying a patch using Patcher via GitHub Actions
+
+Choose a directory named: <CustomizableValue id="DIRECTORY" /> in your repository where we'll add a new infrastructure unit.  In that folder create a file called `terragrunt.hcl` with the following content:
+
+```hcl title="$$DIRECTORY$$/terragrunt.hcl"
+
+terraform {
+  source = "git@github.com:gruntwork-io/terraform-aws-utilities.git//modules/patcher-test?ref=v0.10.3"
+}
+
+# Include the root `terragrunt.hcl` configuration, which has settings common across all environments & components.
+include "root" {
+  path = find_in_parent_folders()
+}
+```
+
+Now we'll push this up to your repository.
+
+```bash
+$ git checkout -b patcher-test-update
+$ git add $$DIRECTORY$$/terragrunt.hcl
+$ git commit -m "Adding a new infrastructure unit that uses the patcher-test module"
+$ git push -u origin patcher-test-update
+
+```
+
+Now navigate to your repository in GitHub and create a new pull request for the `patcher-test-update` branch.  In order for Patcher to identify this unit as needing an update, it first has to be merged to main, so go ahead and merge the pull request.
```

Comment:
NIT: The references to main as the deploy branch here could be a `CustomizableValue` so that the workflow can also be updated if necessary.